<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Url Feed</string>
  </property>
  <widget class="QRadioButton" name="csvRB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Extract from Excel/csv file</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="fileRB">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Extract from another file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browsePB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="browseL">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter separator:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="separatorLE">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="importPB">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPB">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
